wire-ios-data-model/Source/Model/Message/ZMClientMessage+Encryption.swift

```diff
@@ -750,26 +766,32 @@ extension GenericMessage {
         }
 
         let externalGenericMessage = GenericMessage(content: External(withKeyWithChecksum: keys))
+        var encryptedData: Data?
 
-        if DeveloperFlag.proteusViaCoreCrypto.isOn {
-            return externalGenericMessage.encrypt(
-                for: recipients,
-                with: missingClientsStrategy,
-                externalData: data,
-                useQualifiedIdentifiers: useQualifiedIdentifiers,
-                in: context
-            )
-        } else {
-            return externalGenericMessage.legacyEncrypt(
-                for: recipients,
-                with: missingClientsStrategy,
-                externalData: data,
-                useQualifiedIdentifiers: useQualifiedIdentifiers,
-                in: context
-            )
-        }
-
+        ProteusProvider(context: context).perform(
```

Comment:
```suggestion
        context.proteusProvider.perform(
```
just to make it consistent across the code how we get the provider